package-lock.json

```diff
@@ -6016,13 +6241,13 @@
       }
     },
     "@types/inquirer": {
-      "version": "6.0.1",
-      "resolved": "https://registry.npmjs.org/@types/inquirer/-/inquirer-6.0.1.tgz",
-      "integrity": "sha512-O9rEHE9iBvYaFAGS0fAlDzqY/3CsOrRKzni4zwnAEce2JrHUEbXAce2Pwwe8ZGzmQkucwSXn1tSiKig37INgfA==",
+      "version": "6.5.0",
+      "resolved": "https://registry.npmjs.org/@types/inquirer/-/inquirer-6.5.0.tgz",
+      "integrity": "sha512-rjaYQ9b9y/VFGOpqBEXRavc3jh0a+e6evAbI31tMda8VlPaSy0AZJfXsvmIe3wklc7W6C3zCSfleuMXR7NOyXw==",
       "dev": true,
```

Comment:
Why do you update this dependencies?
Consider to use `npm run lerna:bootstrap:ci` unless you really want to install or update something in dependencies.

I would like to suggest to:
- revert the package-lock.json
- manually add the section about `dialog`:
```js
"@tradeshift/elements.dialog": {
      "version": "file:packages/components/dialog",
      "requires": {
        "@tradeshift/elements": "^0.17.0",
        "@tradeshift/elements.icon": "^0.17.0",
        "@tradeshift/elements.modal": "^0.17.0",
        "@tradeshift/elements.typography": "^0.17.0"
      }
},
```
- run `npx lerna bootstrap --ci`